pkg/build/kube/bazelbuildbits.go

```diff
@@ -56,74 +60,112 @@ func (b *BazelBuildBits) Build() error {
 	// make sure we cd back when done
 	defer os.Chdir(cwd)
 
+	// TODO(bentheelder): we assume the host arch, but cross compiling should
+	// be possible now
+	bazelGoosGoarch := fmt.Sprintf("linux_%s", runtime.GOARCH)
+
 	// build artifacts
 	cmd := exec.Command(
 		"bazel", "build",
-		// TODO(bentheelder): we assume linux amd64, but we could select
-		// this based on Arch etc. throughout, this flag supports GOOS/GOARCH
-		"--platforms=@io_bazel_rules_go//go/toolchain:linux_amd64",
-		// we want the debian packages
-		"//build/debs:debs",
+		fmt.Sprintf("--platforms=@io_bazel_rules_go//go/toolchain:%s", bazelGoosGoarch),
+		// node installed binaries
+		"//cmd/kubeadm:kubeadm", "//cmd/kubectl:kubectl", "//cmd/kubelet:kubelet",
 		// and the docker images
-		//"//cluster/images/hyperkube:hyperkube.tar",
 		"//build:docker-artifacts",
 	)
 	exec.InheritOutput(cmd)
 	if err := cmd.Run(); err != nil {
 		return err
 	}
 
+	// capture the output paths
+	b.paths = b.findPaths(bazelGoosGoarch)
+
 	// capture version info
 	return buildVersionFile(b.kubeRoot)
 }
 
-// Paths implements Bits.Paths
-func (b *BazelBuildBits) Paths() map[string]string {
+func (b *BazelBuildBits) findPaths(bazelGoosGoarch string) map[string]string {
 	// https://docs.bazel.build/versions/master/output_directories.html
 	binDir := filepath.Join(b.kubeRoot, "bazel-bin")
 	buildDir := filepath.Join(binDir, "build")
-	return map[string]string{
-		// debians
-		filepath.Join(buildDir, "debs", "kubeadm.deb"):        "debs/kubeadm.deb",
-		filepath.Join(buildDir, "debs", "kubelet.deb"):        "debs/kubelet.deb",
-		filepath.Join(buildDir, "debs", "kubectl.deb"):        "debs/kubectl.deb",
-		filepath.Join(buildDir, "debs", "kubernetes-cni.deb"): "debs/kubernetes-cni.deb",
-		filepath.Join(buildDir, "debs", "cri-tools.deb"):      "debs/cri-tools.deb",
```

Comment:
How are cri-tools and kubernetes-cni now installed? Don't see any references to them elsewhere 🙄

Reply:
they're installed in the base image, this is now on par with the make/bash/docker build.

these debs just wrap downloading a copy of a stable release of these based on WORKSPACE, we do ~the same in the base image already.

Reply:
or rather CNI is:
https://github.com/kubernetes-sigs/kind/blob/6f430d3884438622ec8e9b4483d235c35b8938e4/images/base/Dockerfile#L82-L92
but cri-tools was only to satisfy the deb install requirements, we don't use those while we're on the dockershim path.

Reply:
if/when we take a poke at CRI path we can install these to the base image, they have stable releases and don't change often, and IIRC are backwards/forwards compatible.